src/v2/Components/ArtworkFilter/Utils/urlBuilder.tsx

```diff
@@ -46,18 +50,24 @@ export const buildUrl = (state: ArtworkFilters, pathname?: string): string => {
   return url
 }
 
-export const updateUrl = (state: ArtworkFilters) => {
-  const url = buildUrl(state)
+export const updateUrl = (
+  state: ArtworkFilters,
+  defaultValues?: Partial<ArtworkFilters>
+) => {
+  const url = buildUrl(state, "", defaultValues)
```

Comment:
**minor:** we should be able to write this as `buildUrl(state, defaultValues: defaultValues)` to avoid passing the `""`